iip-template.md

```diff
@@ -0,0 +1,100 @@
+# IIP Guide
+
+## Overview
+
+This guide outlines the structure and key elements of an IIP document, providing a template for authors to use when submitting proposals. Each IIP is identified by a unique number and follows a specific file naming convention.
+
+### File Naming Convention
+
+The file name for an IIP should adhere to the following convention:
+- The file is named with two digits, where a single-digit number is always prepended with a 0.
+- Examples:
+  - 01.md
+  - 02.md
+  - ...
+  - 100.md
+  - 121.md
+
+## IIP Structure
+
+The structure of an IIP document is designed to provide a clear and organized presentation of the proposal. Each IIP consists of the following sections:
+
+### IIP Number
+
+The IIP number is a unique identifier for the proposal and is placed at the beginning of the document, following the format `IIP-XX`, where XX is the assigned number.
+
+Example:
+```markdown
+IIP-XX
+======
+```
+
+### Title
+
+The title section should succinctly convey the main idea or purpose of the proposal.
+
+Example:
+```markdown
+[Title]
+------
+```
+### Authors
+
```

Comment:
for title an other stuff which is not a part of content we can use this style at the top of file:

```yml
---
iip: 0
title: IIP title
status: draft, standard, withdrawn
type: optional, requier
author: name <email@example.com>
created: 2023-07-10
---
```

Reply:
> for title an other stuff which is not a part of content we can use this style at the top of file:
> 
> ```yaml
> ---
> iip: 0
> title: IIP title
> status: draft, standard, withdrawn
> type: optional, requier
> author: name <email@example.com>
> created: 2023-07-10
> ---
> ```

I agree, but I think the `created` is not really needed because it is already version controlled
for `status` I think `draft` and `standard`, if something is `withdrawn` it should removed from the project.

Reply:
that's ok, we can remove them.

but created and withdrawn status help us to have a history.
that's up to you, keep or remove them.

Reply:
Let's keep the date then, we remove it if cumbersome

Reply:
ok, keep the date and draft, standard statuses.